dlc-manager/tests/ln_dlc_channel_execution_tests.rs

```diff
@@ -1568,17 +1567,6 @@ fn off_chain_close_offer(
     bob_node: &LnDlcParty,
     channel_id: ChannelId,
 ) {
-    let sub_channel = alice_node
-        .dlc_manager
-        .get_store()
-        .get_sub_channel(channel_id)
-        .unwrap()
-        .unwrap();
-
-    let dlc_channel_id = sub_channel.get_dlc_channel_id(0).unwrap();
-    let contract_id =
-        assert_channel_contract_state!(alice_node.dlc_manager, dlc_channel_id, Confirmed);
-
```

Comment:
Is there a reason to remove this? I checked and the test still passes with it so I'd rather keep it.

Reply:
I can see the unused `contract_id` generated a warning, but would be better to keep the assert.

Reply:
No reason to remove it other than Clippy. I probably made a mistake. I'll fix it.

Reply:
Yeah, I just deleted it too much D: